execute/plugin_functions.go

```diff
@@ -599,13 +599,13 @@ func getConsensusObservation(
 func getMessageTimestampMap(
 	commitReportCache map[cciptypes.ChainSelector][]exectypes.CommitData,
 	messages exectypes.MessageObservations,
-) (map[cciptypes.Bytes32]time.Time, error) {
+) map[cciptypes.Bytes32]time.Time {
 	messageTimestamps := make(map[cciptypes.Bytes32]time.Time)
 
 	for chainSel, SeqNumToMsg := range messages {
 		commitData, ok := commitReportCache[chainSel]
 		if !ok {
-			return nil, fmt.Errorf("missing commit data for chain %s", chainSel)
+			continue
```

Comment:
Log an error here?
```suggestion
			lggr.Errorf("missing commit data for chain %s, this should not happen if the arguments were initialized correctly") 
```

Reply:
I wonder if this function could also be folded into getAllMessages to avoid all the special setup